include/deal.II/grid/grid_tools.h

```diff
@@ -161,24 +188,16 @@ namespace GridTools
    * collective operation.
    *
    * @param tria The triangulation.
-   * @param mapping An optional argument used to denote the mapping that
-   * should be used when describing whether cells are bounded by straight or
-   * curved faces. The default is to use a $Q_1$ mapping, which corresponds to
-   * straight lines bounding the cells.
+   * @param mapping the Mapping used to compute Jacobians. This is useful when
```

Comment:
```suggestion
   * @param mapping The Mapping used to compute Jacobians. This is useful when
```

Reply:
Maybe say "compute the Jacobians that enter into the evaluation of quadrature weights to approximate the integrals"?

Reply:
I rewrote the whole thing to make it clearer.